stackdriver/lib/stackdriver.rb

```diff
@@ -17,9 +17,9 @@
 gem "google-cloud-logging"
 gem "google-cloud-monitoring"
 
-require "google/cloud/error_reporting/v1beta1"
 require "google/cloud/logging"
-require "google/cloud/monitoring/v3"
+
+require "google/cloud/error_reporting/middleware" if defined? ::Rack
```

Comment:
Users can still load the middleware outside of using the stackdriver gem, so I think it can be appropriate to keep the requires as is.